src/status_im/chat/events/input.cljs

```diff
@@ -28,466 +29,522 @@
+        new-db                (model/set-chat-ui-props db {:selection selection})
+        chat-parameter-box-fx (load-chat-parameter-box new-db (:command command))]
+    (cond-> {:db new-db}
+
+      chat-parameter-box-fx
+      (merge chat-parameter-box-fx)
+
+      (and (= selection (+ (count const/command-char)
+                           (count (get-in command [:command :name]))
+                           (count const/spacing-char)))
+           (get-in command [:command :sequential-params]))
+      (merge (chat-input-focus new-db :seq-input-ref)))))
+
+(defn select-chat-input-command
+  "Selects command + (optional) arguments as input for active chat"
+  [{:keys [current-chat-id chat-ui-props] :as db}
+   {:keys [prefill prefill-bot-db sequential-params name] :as command} metadata prevent-auto-focus?]
+  (let [fx (-> db
+               bots-events/clear-bot-db
+               (model/set-chat-ui-props {:show-suggestions?   false
+                                         :result-box          nil
+                                         :validation-messages nil
+                                         :prev-command        name})
+               (set-chat-input-metadata metadata)
+               (set-chat-input-text (str (chat-utils/command-name command)
+                                         const/spacing-char
+                                         (when-not sequential-params
+                                           (input-model/join-command-args prefill))))
+               (as-> fx'
+                   (merge fx' (load-chat-parameter-box (:db fx') command))))]
+    (cond-> fx
+      prefill-bot-db (update :db bots-events/update-bot-db {:db prefill-bot-db})
+
+      (not (and sequential-params
+                prevent-auto-focus?))
+      (merge (chat-input-focus (:db fx) :input-ref))
+
+      sequential-params
+      (as-> fx'
+          (cond-> (update fx' :db
+                          set-chat-seq-arg-input-text
+                          (str/join const/spacing-char prefill))
+            (not prevent-auto-focus?)
+            (merge fx' (chat-input-focus (:db fx') :seq-input-ref)))))))
+
+(defn set-contact-as-command-argument
+  "Sets contact as command argument for active chat"
+  [db {:keys [bot-db-key contact arg-index]}]
+  (let [name    (str/replace (:name contact) (re-pattern const/arg-wrapping-char) "")
+        contact (select-keys contact [:address :whisper-identity :name :photo-path :dapp?])]
+    (-> (set-command-argument db arg-index name true)
+        (as-> fx
+            (merge fx (bots-events/set-in-bot-db
+                       (:db fx)
+                       {:path [:public (keyword bot-db-key)]
+                        :value contact})))
+        (as-> fx
+            (let [{:keys [current-chat-id]
+                   :as new-db}             (:db fx)
+                  arg-position             (input-model/argument-position new-db)
```

Comment:
WARNING: Wrong number of args (1) passed to status-im.chat.models.input/argument-position at line 244 /home/yenda/status-react/src/status_im/chat/events/input.cljs

Reply:
This is already fixed in #1710 superseding this PR